tensorflow/python/eager/context_test.py

```diff
@@ -136,8 +136,12 @@ def testGetMemoryInfo(self):
 
   @test_util.disable_tfrt('b/169293680: TFE_GetTotalMemoryUsage is unsupported')
   def testGetMemoryInfoCPU(self):
-    with self.assertRaisesRegex(ValueError, 'Allocator stats not available'):
-      context.context().get_memory_info('CPU:0')
+    # MklCPUAllocator (mkl_cpu_allocator.h) does not throw exception.
+    # So skip the test.
+    # TODO(gzmkl) work with Google team to address design issue in allocator.h
+    if not test_util.IsMklEnabled():
```

Comment:
nit:

```
if test_util.IsMklEnabled():
  # TODO(gzmkl): work with Google team to address design issue in allocator.h
  self.skipTest("MklCPUAllocator (mkl_cpu_allocator.h) does not throw exception. So skip the test.")
```

Please update other parts as well.

Reply:
@kkimdev Thank you for the suggestion, based on which I have made code change